### PR TITLE
term: must vs. should

### PR DESCRIPTION
**Pull request by jorgeorpinel**
> Non-extensive in the case of "should" — there's too many instances.

\+ some copy edits around the instances I did review (even if they weren't changed, e.g. a small improvement to the `remote add` ref)

### CODE REVIEW

File: content/docs/command-reference/gc.md [hunk 1]
```diff
@@ -31,9 +31,9 @@ of commits (determined by reading the DVC-files in them). See the
 > Note that `dvc gc` tries to fetch any missing
 > [`.dir` files](/doc/user-guide/dvc-files-and-directories#structure-of-the-cache-directory)
 > from [remote storage](/doc/command-reference/remote) to the local
-> <abbr>cache</abbr>, in order to know which files should exist inside cached
-> directories. These files may be missing if the cache directory was previously
-> garbage collected, in a newly cloned copy of the repo, etc.
+> <abbr>cache</abbr>, in order to determine which files should exist inside
+> cached directories. These files may be missing if the cache directory was
+> previously garbage collected, in a newly cloned copy of the repo, etc.
```
Comment (jorgeorpinel) [inline content/docs/command-reference/gc.md:36]: Unrelated

File: content/docs/command-reference/install.md [hunk 2]
```diff
@@ -276,8 +276,8 @@ featurize:
 
 We see that the output of `dvc status` has appeared in the `git commit`
 interaction. This new behavior corresponds to the Git hook installed, and it
-informs us that the workspace is out of sync. Therefore, we can conclude that
-the `dvc repro` command is needed now:
+informs us that the workspace is out of sync. Therefore, we know that
+`dvc repro` command is needed:
```
Comment (jorgeorpinel) [inline content/docs/command-reference/install.md:280]: Unrealted?

Comment (jorgeorpinel) [reply content/docs/command-reference/install.md:280]: Yes, changes to this whole file are unrelated.

File: content/docs/command-reference/remote/add.md [hunk 3]
```diff
@@ -18,14 +18,21 @@ positional arguments:
...
-or even a directory in the local file system. (See all the supported remote
-storage types in the examples below.) If `url` is a relative path, it will be
-resolved against the current working directory, but saved **relative to the
-config file location** (see LOCAL example below). Whenever possible, DVC will
-create a remote directory if it doesn't exist yet. (It won't create an S3 bucket
-though, and will rely on default access settings.)
+`name` and `url` are required. The `name` is used to identify the remote and
+must be unique for the project.
+
+`url` specifies a location to store your data. It can represent a cloud storage
```
Comment (jorgeorpinel) [inline content/docs/command-reference/remote/add.md:24]: Unrelated (most of the file changes)

File: content/docs/install/macos.md [hunk 4]
```diff
@@ -17,9 +17,9 @@ Get the PKG (binary) from the big "Download" button on the [home page](/), or
 from the [release page](https://github.com/iterative/dvc/releases/) on GitHub.
 
 > Note that currently, in order to open the PKG file, you must go to the
-> Downloads directory in Finder and use
+> **Downloads** directory in Finder and do a
 > [secondary click](https://support.apple.com/en-us/HT207700) on it, then select
-> "Open With" > **Installer.app**, and choose the **Open** button.
+> "Open With" > **Installer.app**, and click the **Open** button.
```
Comment (jorgeorpinel) [inline content/docs/install/macos.md:22]: Unrelated

File: content/docs/user-guide/contributing/blog.md [hunk 5]
```diff
@@ -13,9 +13,9 @@ updating content in your browser as you edit the post.
 
 ## Add a post
 
-Create a Markdown file in the `content/blog` folder. File name must follow the
-pattern: `YYYY-MM-DD-my-new-blog-post.md`. After merging this file into the
-`master` branch, blog post will be published at `blog/my-new-blog-post` address
+Create a Markdown file in the `content/blog` folder. The file name must follow
+this pattern: `YYYY-MM-DD-my-post-title.md`. After merging this file into the
+`master` branch, the blog post will be published at `/blog/my-post-title`
```
Comment (jorgeorpinel) [inline content/docs/user-guide/contributing/blog.md:18]: Unrelated

File: content/docs/command-reference/remote/add.md (hunk 3 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remote/add.md:24]: Well, they were triggered by the "must" instances here. Could be left in the PR, it's a good improvement, I think.

File: content/docs/install/macos.md (hunk 4 above)
Comment (jorgeorpinel) [reply content/docs/install/macos.md:22]: Never mind, it's around a "must" instance.

File: content/docs/command-reference/gc.md (hunk 1 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/gc.md:36]: Actually, this is around "should" so it's somewhat realted. Resolving

File: content/docs/command-reference/install.md (hunk 2 above)
Comment (shcheklein) [reply content/docs/command-reference/install.md:280]: are you going to remove those from this PR?

File: content/docs/user-guide/setup-google-drive-remote.md [hunk 6]
```diff
@@ -211,17 +211,14 @@ these credentials with others**. Each team member should go through this process
...
-with the `gdrive_user_credentials_file` setting:
+`.dvc/tmp/gdrive-user-credentials.json` file. It can be overridden with the
+`gdrive_user_credentials_file` setting:
 
 ```dvc
 $ dvc remote modify myremote gdrive_user_credentials_file \
                     .dvc/tmp/myremote-credentials.json
 ```
 
-> If you edit the config file manually, value must be either an absolute path or
```
Comment (shcheklein) [inline content/docs/user-guide/setup-google-drive-remote.md:222]: why do we remove this?

File: content/docs/user-guide/privacy.md [hunk 7]
```diff
@@ -34,8 +34,8 @@ which requires that you authenticate yourself as a specific Google user and
 authorize these actions.
 
 DVC can help you get a token by guiding you through the OAuth flow in the
-browser. There you must consent to allow the _DVC_ Application to operate on
-your behalf. The OAuth consent screen will describe the scope of what is being
+browser. From there you can allow the _DVC_ Application to operate on your
```
Comment (shcheklein) [inline content/docs/user-guide/privacy.md:37]: let's not change this legal language

File: content/docs/user-guide/privacy.md [hunk 8]
```diff
@@ -78,9 +78,9 @@ your application must accurately represent itself when authenticating to Google
 API services.
 
 If you use DVC inside another package or application that executes its own logic
-— as opposed to code in DVC or by the user — you must communicate this clearly
+— as opposed to code in DVC or by the user — you should communicate this clearly
```
Comment (shcheklein) [inline content/docs/user-guide/privacy.md:81]: same here, let's not change legal language

File: content/docs/command-reference/remote/modify.md [hunk 9]
```diff
@@ -218,8 +218,8 @@ these settings, you could use the following options:
 To communicate with a remote object storage that supports an S3 compatible API
 (e.g. [Minio](https://min.io/),
 [DigitalOcean Spaces](https://www.digitalocean.com/products/spaces/),
-[IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) etc.) you
-must explicitly set the `endpointurl` in the configuration:
+[IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) etc.)
+please set the `endpointurl` explicitly in the configuration:
```
Comment (shcheklein) [inline content/docs/command-reference/remote/modify.md:222]: `please` - we don't usually use that kind of language, especially in the command reference

File: content/docs/command-reference/install.md (hunk 2 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/install.md]: Yes, sorry. Gone now.

File: content/docs/user-guide/setup-google-drive-remote.md (hunk 6 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/setup-google-drive-remote.md:222]: Big conceptual jump from `dvc remote modify` to "config file" which isn't explained in this doc. I.e. I don't think users will understand the note at all. We'd need to expand on that relationship and I don't think it's worth it. But lmk please

File: content/docs/user-guide/privacy.md (hunk 7 above)
Comment (jorgeorpinel) [reply content/docs/user-guide/privacy.md:37]: OK, changes undone. Should I avoid touching this file completely going fwd?

File: content/docs/command-reference/remote/modify.md (hunk 9 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remote/modify.md]: OK, updated. But there's 23 more instances of the word in the cmd ref, actually 🙂

File: content/docs/command-reference/remote/modify.md [hunk 10]
```diff
@@ -236,8 +236,8 @@ these settings, you could use the following options.
 To communicate with a remote object storage that supports an S3 compatible API
 (e.g. [Minio](https://min.io/),
 [DigitalOcean Spaces](https://www.digitalocean.com/products/spaces/),
-[IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) etc.) you
-must explicitly configure the `endpointurl`:
```
Comment (shcheklein) [inline content/docs/command-reference/remote/modify.md:240]: not sure about this one ... you must do this indeed

File: content/docs/command-reference/remote/add.md [hunk 11]
```diff
@@ -128,8 +128,8 @@ So, make sure you have the following permissions enabled:
 To communicate with a remote object storage that supports an S3 compatible API
 (e.g. [Minio](https://min.io/),
 [DigitalOcean Spaces](https://www.digitalocean.com/products/spaces/),
-[IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) etc.) you
-must explicitly configure the `endpointurl`:
+[IBM Cloud Object Storage](https://www.ibm.com/cloud/object-storage) etc.)
+please explicitly configure the `endpointurl`:
```
Comment (shcheklein) [inline content/docs/command-reference/remote/add.md:132]: same here

**Comment by jorgeorpinel** [comment]
Thanks! Ah yes, I forgot that IBM... phrase is repeated in several places... Fixing ⌛

File: content/docs/command-reference/remote/modify.md (hunk 10 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remote/modify.md:240]: Yes, that's what this says now (it's just a long sentence): "To communicate with ... an S3 compatible API ... configure the remote's `endpointurl` explicitly". The word must is very emphatic, I don't think we needed it here. It's still imperative sentence, so it tells you what you have to do, basically.

File: content/docs/command-reference/remote/add.md (hunk 11 above)
Comment (jorgeorpinel) [reply content/docs/command-reference/remote/add.md]: Fixed!
